lib/tasks/evm_automate.rake

```diff
@@ -10,7 +10,10 @@ module EvmAutomate
   end
 
   def self.simulate(domain, namespace, class_name, instance_name)
+    user = User.where(:userid => 'admin').first
```

Comment:
@gmcculloug @tinaafitz can we get rid of this rake call and have user just use the ui? Who can make this call?

a) yes -->  cool. that simplifies that via delete.
b) no --> we'll have to change the cli to allow the user to pick a user. (which means userid / group / tenant)
(this can be a separate PR, but it really needs to be in there)